system-tests/projects/no-specs-no-storybook/cypress.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  component: {
+    supportFile: 'cypress/component/support.js',
+    devServer: require('@cypress/react/plugins/load-webpack'),
+    devServerConfig: {
+      webpackFilename: 'webpack.config.js',
+    },
+  },
+  e2e: {
+    supportFile: 'cypress/e2e/support.js',
```

Comment:
These support files are empty. Could we just do the follow to add less files?
```suggestion
    supportFile: false,
```

Reply:
At one point this was necessary, but I think I can remove these at this point.